addons/viewdistance/stringtable.xml

```diff
@@ -114,7 +114,7 @@
             <Portuguese>Distância de visão dinâmica dos objetos</Portuguese>
         </Key>
         <Key ID="STR_ACE_ViewDistance_object_Description">
-            <English>Sets the object view distance as a coefficient of the view distance.</English>
+            <English>Sets the object view distance as a coefficient of the view distance or based on field of view. FoV Based's lowest and highest value is the value which is set when enabling the option.</English>
```

Comment:
In case you're looking for translations, this is what it should it be for German:

> Passt die Objektsichtweite dynamisch der Sichtweite an oder setzt sie auf Grundlage des Sichtfeldes (FoV). 
> Wenn in Abhängigkeit vom Sichtfeld, dann wird der für "Objektsichtweite" eingestellte Wert das Minimum, und die eingestellte "Sichtweite" das Maximum der Objektsichtweite.